code/modules/tgui/modules/communications.dm

```diff
@@ -104,9 +104,10 @@
 		"line_2" = (stat_msg2 ? stat_msg2 : "-----"),
 
 		"presets" = list(
-			list("name" = "blank",    "label" = "Clear",       "desc" = "Blank slate"),
-			list("name" = "shuttle",  "label" = "Shuttle ETA", "desc" = "Display how much time is left."),
-			list("name" = "message",  "label" = "Message",     "desc" = "A custom message.")
+			list("name" = "blank",    "label" = "Clear",        "desc" = "Blank slate."),
+			list("name" = "time",     "label" = "Station Time", "desc" = "The current time according to the station's clock."),
+			list("name" = "shuttle",  "label" = "Shuttle ETA",     "desc" = "Display how much time is left."),  //CHOMPEdit we have a shuttle not a tram silly virgo//VOREStation Edit - Shuttle ETA -> Tram ETA because we use trams
```

Comment:
Do we need to keep the original Virgo comment?

Reply:
Yea, it's there to remind us why virgo changed it in the first place and why we needed to change it back.